examples/config/pushpin.conf

```diff
@@ -19,7 +19,7 @@ stats_connection_send=true
 
 [runner]
 # services to start
-services=condure,zurl,pushpin-proxy,pushpin-handler
```

Comment:
Is this intentional in this commit?

Reply:
Ah yes, I forgot to mention: since we will always prefer our bundled condure, we can assume it has all the features we need, including its outbound connection support which supersedes zurl.